web-console/monitoring/alertmanager-config/alertmanager-config.md

```diff
@@ -0,0 +1,32 @@
+# Alertmanager Configuration
+
+![alerts list view](img/alerts-list.png)
+- To consolidate all admin-level alerting functions together, `Alerts` and `Silences` pages could be combined onto the same page as tabs with the `Configuration` and `YAMl` pages
+
+![alerting configuation tab](img/config-empty.png)
```

Comment:
Agree about potentially adding info popovers – one thing that was not immediately apparent to me in the "group by" section was that the alerts would be grouped in the case that it meets any of the criteria (not a combination of it).